sc2reader/factories/plugins/replay.py

```diff
@@ -100,21 +100,30 @@ def APMTracker(replay):
     above actions divided by the number of seconds played by the player (not
     necessarily the whole game) multiplied by 60.
     """
+
+    speed_multiplier = 1
+    if replay.expansion == 'LotV':
+        speed_multiplier = 1.4
```

Comment:
how about consolidating all the `1.4`s into a single place?  i see them in `sc2reader/engine/plugins/apm.py`, `sc2reader/factories/plugins/replay.py` and  `sc2reader/resources.py`